ethstorage/miner/worker.go

```diff
@@ -241,12 +240,25 @@ func (w *worker) newWorkLoop() {
 				go w.taskLoop(taskCh)
 			}
 			w.lg.Info("Worker is starting task loops", "shard", shardIdx, "threads", w.config.ThreadsPerShard)
+
+			if w.config.EmailConfig.Enabled {
```

Comment:
This behavior is inconsistent with the guide, as it sends an email every time the es-node restarts after syncing.

Reply:
Guide updated.